packages/editor/src/client/blocks/richTextBlockTable.less

```diff
@@ -0,0 +1,16 @@
+table {
+  white-space: pre-wrap;
+  width: 100%;
+  margin: 10px;
```

Comment:
I'm not sure if `px` is the best unit for `margin` and `padding` since it can look ugly with different screen resolutions. `em` and `rem` would be alternatives. What do you think @roosnic1 ?

Reply:
https://wepublish.atlassian.net/browse/WEP-118